[DOCS] Case file attachments (#3236)
elastic/security-docs: docs/cases/cases-manage.asciidoc

[role="screenshot"]
image::images/cases-alert-tab.png[Shows you the Alerts tab]

[float]
[[cases-add-files]]
=== Add files

To upload files to a case, click the *Files* tab:

[role="screenshot"]
image::images/cases-files.png[A list of files attached to a case]
// NOTE: This is an autogenerated screenshot. Do not edit it directly.

> The acceptable file types and sizes are affected by your {kibana-ref}/cases-settings.html[{kib} case settings].

Reviewer: ```suggestion
You can set file types and sizes by configuring your {kibana-ref}/cases-settings.html[{kib} case settings].
```
